xchange-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -0,0 +1,148 @@
+package info.bitrich.xchangestream.kraken;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import info.bitrich.xchangestream.kraken.dto.enums.KrakenEventType;
+import info.bitrich.xchangestream.kraken.dto.KrakenSubscriptionConfig;
+import info.bitrich.xchangestream.kraken.dto.KrakenSubscriptionMessage;
+import info.bitrich.xchangestream.kraken.dto.enums.KrakenSubscriptionName;
+import info.bitrich.xchangestream.kraken.dto.KrakenSubscriptionStatusMessage;
+import info.bitrich.xchangestream.kraken.dto.KrakenSystemStatus;
+import info.bitrich.xchangestream.service.netty.JsonNettyStreamingService;
+import info.bitrich.xchangestream.service.netty.StreamingObjectMapperHelper;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static info.bitrich.xchangestream.kraken.dto.enums.KrakenEventType.subscribe;
+
+/**
+ * @author makarid, pchertalev
+ */
+public class KrakenStreamingService extends JsonNettyStreamingService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(KrakenStreamingService.class);
+    private static final String EVENT = "event";
+    private final Map<Integer, String> channels = new HashMap<>();
+    private ObjectMapper mapper = StreamingObjectMapperHelper.getObjectMapper();
+
+    private final Map<Integer, String> subscriptionRequestMap = new HashMap<>();
+
+    public KrakenStreamingService(String apiUrl) {
+        super(apiUrl, Integer.MAX_VALUE);
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
This is modifying a shared `ObjectMapper` which is dangerous.

In any case, it's not necessary. I think this is the default.

Reply:
Removed this config